doc/nrf/libraries/modem/uicc_lwm2m.rst

```diff
@@ -0,0 +1,28 @@
+.. _lib_uicc_lwm2m:
+
+UICC LwM2M
+##########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The UICC LwM2M library provides functionality to read LwM2M bootstrap configuration from SIM.
+
+Dependencies
```

Comment:
Could move dependencies section before the "API documentation" section as per the template here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/library_template_README.html.